Twig/Extension/LadybugExtension.php

```diff
@@ -34,8 +34,8 @@ public function __construct(Dumper $ladybug)
     public function getFilters()
     {
         return array(
-            'ladybug_dump' => new \Twig_Filter_Method($this, 'ladybug_dump', array('is_safe' => array('html'))),
-            'ld'  => new \Twig_Filter_Method($this, 'ladybug_dump', array('is_safe' => array('html')))
+            new \Twig_SimpleFilter('ladybug_dump',array($this, 'ladybug_dump', array('is_safe' => array('html')))),
```

Comment:
This is wrong. `array($this, 'ladybug_dump')` should be callable and second paramater  and `array('is_safe' => array('html'))` should be the third parameter of `Twig_SimpleFilter`.